frontend/packages/console-dynamic-plugin-sdk/src/app/redux.ts

```diff
@@ -1,5 +1,5 @@
-import coreReducer from './core/reducers/core';
+import { coreReducer } from './core/reducers/core';
 
-export const baseReducers = Object.freeze({
+export const SDKReducer = Object.freeze({
```

Comment:
```suggestion
export const SDKReducers = Object.freeze({
```
When someone imports our reducers, we don't want them thinking that it's a singular one (or a combined set) that will take a single prop. This is our "list" of our internal reducers... they should just spread these forward and treat it opaque-like.

Maybe we should put a JS Doc comment here. Something like:

```
/**
 * Dynamic Plugin SDK Redux store reducers
 *
 * If you use Redux, these can be spread into the root of your store to provide an integrated SDK.
 * If you do not use Redux, these will be provided via the SDK Redux Store.
 */
export const SDKReducers = Object.freeze({
```